config/distribution/linux_cu101.cmake

```diff
@@ -35,4 +35,4 @@ set(USE_F16C OFF CACHE BOOL "Build with x86 F16C instruction support")
 set(USE_LIBJPEG_TURBO ON CACHE BOOL "Build with libjpeg-turbo")
 
 set(CUDACXX "/usr/local/cuda-10.1/bin/nvcc" CACHE STRING "Cuda compiler")
-set(MXNET_CUDA_ARCH "3.0;5.0;6.0;7.0;7.5" CACHE STRING "Cuda architectures")
+set(MXNET_CUDA_ARCH "3.0;5.0;6.0;7.0" CACHE STRING "Cuda architectures")
```

Comment:
why remove 7.5?

Reply:
Size limitation

Reply:
Well why not remove another one then which has a smaller use base? This is about precompiling kernels for convenience, so I'd rather skip a less frequently used one

Reply:
Code compiled for 7.0 runs fine on 7.5

Reply:
Right, but it causes an increased initial startup time while the kernels are generated on the fly, doesn't it?

Reply:
No. But compiling for 7.5 can result in faster runtime. So we should definitely fix the size limitation issues and re-enable the 7.5 target once possible.
See  https://docs.nvidia.com/cuda/turing-compatibility-guide/index.html#turing-volta-compatibility